dbc/js/flags.js

```diff
@@ -209,6 +209,19 @@ const mapFlags = {
 	0x4000000: 'GARRISON'
 }
 
+const soundkitFlags = {
+	0x0001: 'UNK1',
```

Comment:
Fyi: site is fine with missing values, no need to add completely unknown ones :)